docs/api/manual/Flash.md

```diff
@@ -0,0 +1,136 @@
+    data.tag[0] = 0x6a;
+    data.tag[1] = 0x61;
+    data.tag[2] = 0x6e;
+
+    /* Erase the sector first */
+    erase_sector(address);
+    /* And write the flash */
+    program_flash(address, (char*)&data, sizeof(data));
+
+    printf("Writing to flash OK...\r\n");
+
+    /* Retrieving the data structure from flash */
+    MyFlashStorage* data_from_flash = (MyFlashStorage*)address;
+
+    printf("Read from flash. magic=%d, tag=%c%c%c\r\n", data_from_flash->magic,
+        data_from_flash->tag[0], data_from_flash->tag[1], data_from_flash->tag[2]);
+
+    // ... Rest of your code
+```
+
+When you run this program uVisor immediately throws a bus fault during startup (for information on how to see uVisor log messages, see the [debugging](../DEBUGGING.md) page), because our box does not have access to the peripheral at address `0x40020007`.
+
+```
+***********************************************************
+                    BUS FAULT
+***********************************************************
+
+* FAULT SYNDROME REGISTERS
+
+  CFSR: 0x00008200
+  BFAR: 0x40020007
+  --> PRECISERR: precise data access.
+
+* No MPU violation found
+
+* MEMORY MAP
+  Address:           0x40020007
+  Region/Peripheral: [not available]
+    Base address:    0x40020007
+    End address:     0x40020007
+    AIPS slot:       32
+
+HALT_ERROR(./core/vmpu/src/kinetis/vmpu_kinetis.c#144): Access to restricted resource denied
+```
+
+## Configuring the ACL for the FTFE driver
+
+To find out which peripheral is responsible for this fault, you can look in the [peripheral access layer](https://github.com/ARMmbed/mbed-os/blob/e5ba1d2/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/device/MK64F12.h) for your development board. If you search here for `40020000` - peripherals are normally aligned - you find the `FTFE_BASE` address, which is the flash driver for the K64F.
+
+```cpp
+/** Peripheral FTFE base address */
+#define FTFE_BASE                                (0x40020000u)
+```
+
+You can enable this peripheral for the secure box by adding the following definition to the ACL list:
+
+```cpp
+    {FTFE,             sizeof(*FTFE),  UVISOR_TACLDEF_PERIPH},
+```
+
+After re-compiling the code, you can now see - over the serial port - that you can successfully write to flash. However, uVisor still halts when trying to read from flash.
+
+```
+***********************************************************
+                    BUS FAULT
+***********************************************************
+
+* FAULT SYNDROME REGISTERS
+
+  CFSR: 0x00008200
+  BFAR: 0x000FF000
+  --> PRECISERR: precise data access.
+
+* MPU FAULT:
+  Slave port:       0
+  Address:          0x000FF000
+  Faulting regions:
+    R00: 0x00000000 0xFFFFFFFF 0x000007D0 0x00000001
+  Master port:      0
+  Error attribute:  Data READ (user mode)
+
+* MEMORY MAP
+  Address:           0x000FF000
+  Region/Peripheral: [not available]
+    Base address:    0x000FF000
+    End address:     0x000FF000
```

Comment:
We don't show the MEMORY MAP section anymore, so this section should be removed.